ark/segmentation/marker_quantification_test.py

```diff
@@ -1,32 +1,38 @@
 import numpy as np
-import xarray as xr
 import os
 import pytest
 import tempfile
 
 import skimage.morphology as morph
 from skimage.morphology import erosion
 
-from mibidata import mibi_image as mi, tiff
-
 from ark.segmentation import marker_quantification
-from ark.utils import data_utils_test
-
-from ark.utils.segmentation_utils_test import _create_test_extraction_data
+from ark.utils import test_utils
 
 
 def test_compute_marker_counts():
-    cell_mask, channel_data = _create_test_extraction_data()
-
-    # create xarray containing segmentation mask
-    coords = [range(40), range(40), ['whole_cell']]
-    dims = ['rows', 'cols', 'compartments']
-    segmentation_masks = xr.DataArray(np.expand_dims(cell_mask, axis=-1), coords=coords, dims=dims)
-
-    # create xarray with channel data
-    coords = [range(40), range(40), ['chan0', 'chan1', 'chan2', 'chan3', 'chan4']]
-    dims = ['rows', 'cols', 'channels']
-    input_images = xr.DataArray(channel_data, coords=coords, dims=dims)
+    fovs = ['fov1']
+    chans = [f'chan{i}' for i in range(5)]
+
+    cell_mask, channel_data = test_utils.create_test_extraction_data()
+    cell_mask = np.expand_dims(cell_mask, axis=0)
+    channel_data = np.expand_dims(channel_data, axis=0)
+
+    segmentation_masks = test_utils.make_labels_xarray(
+        label_data=np.expand_dims(cell_mask, axis=-1),
+        fov_ids=fovs,
+        row_size=40,
+        col_size=40,
+        compartment_names=['whole_cell']
+    )[0]
```

Comment:
This [0] is mysterious. I think for ease of understanding this should be on its own line with a comment indicating that the test function produces 4D data but the function being tested takes 3D data